app/views/layouts/application.html.erb

```diff
@@ -20,7 +20,7 @@
         <%= content_tag :p, "Logged in as #{current_user.name}" %>
       <% elsif current_admin? %>
         <%= link_to "Admin's Dashboard", "/admin" %>
-        <%= link_to "All Users", "/admin/users" %>
+        <%= link_to "Users", "/admin/users" %>
```

Comment:
I have been awaiting this moment (where we have a page to see all users) for ages! I'm excited we can do this now (as an admin user)